kuma/javascript/src/header/header.jsx

```diff
@@ -0,0 +1,142 @@
+import gettext from '../gettext.js';
+
+const styles = {
+    headerRow: css({ borderTop: '5px solid #83d0f2' })
+};
+
+const Row = styled.div`
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+`;
+
+const Spring = styled.div`
+    flex: 1 1 0px;
+`;
+
+const Strut = styled.div(props => ({
+    flexGrow: 0,
+    flexShrink: 0,
+    flexBasis: props.width
+}));
+
+// TODO: the labels all need to be localized
+const menus = [
+    {
+        label: 'Technologies',
+        items: [
+            { url: 'Web/HTML', label: 'HTML' },
+            { url: 'Web/CSS', label: 'CSS' },
+            { url: 'Web/JavaScript', label: 'JavaScript' },
+            { url: 'Web/Guide/Graphics', label: 'Graphics' },
+            { url: 'Web/HTTP', label: 'HTTP' },
+            { url: 'Web/API', label: 'APIs / DOM' },
+            {
+                url: 'Mozilla/Add-ons/WebExtensions',
+                label: 'Browser Extensions'
+            },
+            { url: 'Web/MathML', label: 'MathML' }
+        ]
+    },
+    {
+        label: 'References & Guides',
+        items: [
+            { url: 'Learn', label: 'Learn web development' },
+            { url: 'Web/Tutorials', label: 'Tutorials' },
+            { url: 'Web/Reference', label: 'References' },
+            { url: 'Web/Guide', label: 'Developer Guides' },
+            { url: 'Web/Accessibility', label: 'Accessibility' },
+            { url: 'Games', label: 'Game development' },
+            { url: 'Web', label: '...more docs' }
+        ]
+    },
+    {
+        label: 'Feedback',
+        items: [
+            {
+                url: 'https://support.mozilla.org/',
+                label: 'Get Firefox help',
+                external: true
+            },
+            {
+                url: 'https://stackoverflow.com/',
+                label: 'Get web development help',
+                external: true
+            },
+            { url: 'MDN/Community', label: 'Join the MDN community' },
+            {
+                label: 'Report a content problem',
+                external: true,
+                url: `https://github.com/mdn/sprints/issues/new?template=issue-template.md&projects=mdn/sprints/2&labels=user-report&title=${encodeURIComponent(
+                    window.location
+                )}`
+            },
+            {
+                label: 'Report a bug',
+                external: true,
+                url: 'https://bugzilla.mozilla.org/form.mdn'
+            }
+        ]
+    }
+];
+
+function fixurl(url) {
+    if (url.startsWith('https://')) {
+        return url;
+    } else {
+        let locale = window.location.pathname.split('/')[1];
+        return `/${locale}/docs/${url}`;
+    }
+}
+
+const HOME_URL =
+    window && window.location
+        ? `${window.location.protocol}//${window.location.host}/${
+              window.location.pathname.split('/')[1]
+          }/`
+        : 'https://developer.mozilla.org/en-US/';
```

Comment:
In the Django version, ``url(home)`` renders to the path ``/en-US/``, or whatever the current locale is, without the protocol and domain. I think leaving the domain out could make offline rendering for different environments easier.

Reply:
Good point. Fixed.